cat/CaT-service.yaml

```diff
@@ -4153,12 +4150,102 @@ paths:
           $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/500InternalServerError'
         '502':
           $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/502BadGateway'
-      security:
-        - OAuth2:
-            - buyer
       summary: Get contract details
 
+
+  /public/projects:
```

Comment:
Needs to be sat under Tenders. Can I suggest /tenders/public/projects. Better still it should be /tenders/projects and give the appropriate response based on having or not having a token

Reply:
Addressed as part of the PR - https://github.com/Crown-Commercial-Service/ccs-scale-api-definitions/pull/224